### PR TITLE
Morph ambush attack

### PR DESCRIPTION
**Pull request by FriendlyContractor**
## About The Pull Request
Adds morph ambush attack. When somebody attempts to interact with disguised morph unarmed they will be knocked down and injected with powerfull venom which very quickly incapitates them. ~~Also examining disguised morph up close no longer reveals that its a morph~~

## Why It's Good For The Game
Makes morph more interesting by encouraging smart use of disguise in attempt to trick others to interact with it. You can pretend to be a vending machine, computer console, insulated gloves etc and laugh when somebody gets bitten

## Changelog
:cl:
add: Morph will perform powerfull ambush attack on players tricked into interacting with it
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
I really hope this makes it in. Morph has felt horribly weak for a very long time.

**Comment by RuneQuester** [comment]
This seems horribly unbalanced. Why remove the examine text? Why the venom that incapacitates? Does that sleep or stun? This just seems like an unfun situation of "click something random without having any chance to know its a morph, get suddenly hit with a heavy damage chem, die without having any counter". 

Keep the examine text and remove the chem

**Comment by FriendlyContractor** [comment]
First of all to give this change some context - all morphs i have seen (not just recently) first go for the armory to eat its guns and optionally ore silo and then end up "meming" around as a nuke, toilet or donut being completly friendly and you often even get yelled at for killing a "friendly morph". I yet have to see a single person killed by morph and i cannot even recall a situation when morph did as much as ATTACKED somebody. Morph does not have very good combat stats and is incapable of taking somebody out before they can get away to safety which largely prevents him from antagonizing crew members as he will be ganged up on and toolboxed to death if he attempts to pursue. I dont think this makes for a very interesting antagonist

Now to adress you concerns.
- **Why to remove examine text ?** - I dont want morphs ambush to be discovered simply because somebody examined an item out of habit or just randomly. If you suspect something is not what it seems you can just hit it with anything else than your bare hand to check. You should discover a morph by noticing something is out of place such as insulated gloves lying in middle of hallway or vending machine that wasnt there before, not by randomly examining stuff
- **Why the venom that incapacitates ?** - It incapiciates and very quickly so because if somebody can just easily run away to safety of other players after ambush its made completly useless. Intention is that if you get bitten only others can help you in this situation and otherwise you are doomed. Even with all that i suspect sucesfull ambush which results in person being absorbed by morph will be rare sight

**Comment by ItsTisme** [comment]
I think you do need SOME sort of way of descerning a morth, also anything thats a "lol ded" isant fun to play with unless you start shooting **EVERYTHING**

**Comment by francinum** [comment]
These issues are more about morph being kinda meh in general.

Leave the examine tell for now, we can cut that separately. *Do not bundle changes like this together*

**Comment by FriendlyContractor** [comment]
Im losing hope somebody is going to bother reviewing my 30 lines of code ;-;

File: code/modules/antagonists/morph/morph.dm [hunk 1]
```diff
@@ -321,6 +321,17 @@
 			eat(I)
 			return
 	return ..()
+//Ambush attack
+/mob/living/simple_animal/hostile/morph/attack_hand(mob/living/carbon/human/M)
+	if(morphed)
+		M.Knockdown(40)
+		M.reagents.add_reagent(/datum/reagent/toxin/morphvenom, 7)
```
Comment (zeskorion) [inline code/modules/antagonists/morph/morph.dm:328]: the venom isnt really necessary, IMO. make it a paralyze instead, if you want. maybe also give it, like, an actual damaging/attack effect?

**Review by zeskorion — COMMENTED** [review]
maybe, instead of flat out removing the examine text, add it in plain black text to the end of the examine text, instead of as red text- so it's hard to spot even when shift-clicking

File: code/modules/antagonists/morph/morph.dm [hunk 2]
```diff
@@ -321,6 +321,17 @@
 			eat(I)
 			return
 	return ..()
+//Ambush attack
+/mob/living/simple_animal/hostile/morph/attack_hand(mob/living/carbon/human/M)
+	if(morphed)
```
Comment (Archanial) [inline code/modules/antagonists/morph/morph.dm:326]: make it have some sort of cooldown too

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/morph/morph.dm (hunk 2 above)
Comment (FriendlyContractor) [reply code/modules/antagonists/morph/morph.dm:326]: I believe its not needed since shapeshifting already has a cooldown

**Review by Archanial — APPROVED** [review]

**Pull request merged**